model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -46,6 +48,7 @@ class LicenseInfoResolver(
 ) {
     private val resolvedLicenseInfo: ConcurrentMap<Identifier, ResolvedLicenseInfo> = ConcurrentHashMap()
     private val resolvedLicenseFiles: ConcurrentMap<Identifier, ResolvedLicenseFileInfo> = ConcurrentHashMap()
+    private val rootLicenseMatcher = RootLicenseMatcher(rootLicenseFilenamePatterns = emptyList())
```

Comment:
Why do you set the patterns to an empty list?

Reply:
Because readme files must not be put into the notices.

Reply:
I think we need to work on the naming.